### ISSUE TITLE
When a player with any of the Timebomb effects is killed, the bomb should be exploded anyway instead of disappearing

### ISSUE THREAD

**Issue by Tiagoquix** [Phil25/RTD#96]

**Comment by Phil25** [comment]
I have a problem with adding this. Usually when people want to configure perks themselves there is a clear visual indicator to how they are configured, but not sure what to add here.

My question is, how to let random players know whether this particular timebomb will explode anyway, as opposed to the default behavior which makes it safe to get up close to destroy it?

**Comment by Phil25** [comment]
I did refer to Timebomb and Fire Timebomb effects too.

Consider this scenario:
You're playing TF2 and somebody on the other team rolls Timebomb and begins to run at you. Currently, whether you kill the other player from a distance or up close doesn't really matter, you know you're safe as long as you get the kill. However, with this change, server operators will be able to configure Timebomb to explode regardless of the perk timer, meaning that, as a player, you have no way of telling whether you're safe up close and you have to keep your distance just to be safe.

That's generally my problem with this suggestion -- players are unable to tell which setting the server operator went with. And it's crucial for them to know at a glance, because it affects their stategy.

Thinking about this now, I'd approach it like this: if a player with Timebomb is killed, the bomb drops to the ground and runs its time regardless of the owner's death. It explodes normally when it reaches 0.

**Comment by Phil25** [comment]
> Your new way of thinking (dropping the bomb on the ground when killed) is good, but I think it ends up returning to the same problem as the players' uncertainty that you mentioned.

I disagree, players seeing the bomb being dropped to the ground with the exact same effects going on is a clear indicator that it's going to detonate regardless of the owner's death, in my opinion.

> I think perhaps the best way to buff Timebomb perks is to reduce the time until the explosion (reduce effect time), and effect-wise keep it as is. 5 seconds is fine IMO.

Changing the timer doesn't buff or nerf it in any way. With a shorter timer players can utilize the bomb quicker if the situation allows it, with a longer timer they have more time to wait for a good opportunity and choose to attack themselves. That said, I think 10 seconds is fine as it is.
